site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md

```diff
@@ -0,0 +1,115 @@
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
+
+When viewing trace details, a span might be missing in the Checkly visualization. 
+Using contextual span data, about the trace it belongs to, and it's parent span; Checkly displays any missing pieces on the span chain, so that you can identify which application or service might be incorrectly instrumented. 
+
+![Trace with missing spans](/docs/images/otel/traces-missing-spans.png)
+
+
+## Troubleshooting missing spans
+
+Missing spans can happen due to various reasons.  Wether you're sending data to Checkly directly from your applications and services, or through the OpenTelemetry Collector, these are steps to confirm everything is properly setup.
+
+### Verify sampling strategy
+
+   Using Checkly Traces, we recommend sampling for the tracestate header `checkly=true`, which will reduce your Egress/Ingress costs and ensure that all the spans that were originated through a check are there.
+
+   #### Sampling in the OpenTelemetry Collector
+
+      a. Make sure that you're filtering out non-checkly spans:
+
+      ```yaml
+      processors:
+         batch:
+         filter/checkly:
+            error_mode: ignore
+            traces:
+               span:
+               # filter out spans in which tracestate is not "checkly=true"
+               - 'trace_state["checkly"] != "true"'
+      ``` 
+      b. Confirm that your Collector pipeline uses this processor:
+
+      ```yaml
+      service:
+         pipelines:
+            traces:
+               processors: [filter/checkly, batch]
+      ```
+
+   #### Sampling directly in your applications' code. 
+   Choose the specific [instrumentation language guide](docs/traces-open-telemetry/instrumenting-code/) and review your configuration against Step 2 in the guides.
+
+### Review OpenTelemetry Exporter configuration
+  
+   Ensure the right Authorization keys and endpoints are in use.
+
+   #### Exporting through the OpenTelemetry Collector
+      
+   * **Step 1**. Make sure that your exporter uses the right endpoint authorization:
+
+      ```yaml
+      exporters:
+         otlp/checkly:
+            endpoint: "otel.eu-west-1.checklyhq.com:4317"
+            headers:
+               authorization: "${env:CHECKLY_OTEL_API_KEY}"
+      ``` 
+   * **Step 2**. Confirm that your Collector pipeline uses this exporter:
+
+      ```yaml
+      service:
+         pipelines:
+            traces:
+               exporters: [otlp/checkly]
+      ```
+
+   #### Exporting directly from your Application's code
+      Confirm the environment variables use the right endpoint and authorization:
+
+      ```bash
```

Comment:
The formatting didn't work properly here
<img width="702" alt="Screenshot 2024-10-24 at 08 19 30" src="https://github.com/user-attachments/assets/0c015cd7-cdc9-48b0-8cbb-063beda66cf1">

Reply:
Aha, indentation, thanks!